GUI/MainRepo.cs

```diff
@@ -64,7 +66,22 @@ private void UpdateRepo(object sender, DoWorkEventArgs e)
             try
             {
                 AddStatusMessage("Updating repositories...");
-                e.Result = Repo.UpdateAllRepositories(RegistryManager.Instance(CurrentInstance), CurrentInstance, Manager.Cache, GUI.user);
+
+                // Note the current mods' compatibility for the NewlyCompatible filter
+                KspVersionCriteria versionCriteria = CurrentInstance.VersionCriteria();
+                IRegistryQuerier registry = RegistryManager.Instance(CurrentInstance).registry;
+                Dictionary<string, bool> oldModules = registry.Available(versionCriteria)
+                    .ToDictionary(m => m.identifier, m => false);
```

Comment:
It's false for every mod in `Available` and true for every mod in `Incomaptible.`
There is no `m.IsIncompatible` yet because `m` is a `CkanModule`, not a `GUIModule`.